tests/ingredient.test.ts

```diff
@@ -46,7 +44,9 @@ describe('createIngredient()', () => {
         'contentauth:urn:uuid:699750af-e07b-4c45-9d24-a131442111b8',
       );
       expect(ingredient.manifest_data?.format).toEqual('application/c2pa');
-      expect(ingredient.manifest_data?.identifier).toMatch(identifierMatcher);
+      expect(ingredient.manifest_data?.identifier).toMatch(
+        'manifest_data.c2pa',
+      );
```

Comment:
Very small detail - just two points:
1. If this is no longer a regular expression, you can use `toEqual` instead of `toMatch`
2. Since `manifest_data.c2pa` is used here and on L85, why not keep it in a `const`?

Reply:
All set.